.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,3 @@
+We welcome Pull Requests from Engineers that are working at a startup, which is also the target audience of this document. Try to keep it short and to the point. There is also some tension between keeping it practical, and keeping it vendor neutral. We're open for suggestions.
+
+Thanks for Shahar Keidar, Avishai Ish-Shalom, Yogev Levi Shaked, Elad Shulman and Eliran Ben-Zikri for reviewing, commenting and helping out with writing the first version of this blog post.
```

Comment:
This repository is not blog post.

Reply:
It also was a [blog post](https://tech.forter.com/security-101-for-saas-startups/) that was double posted

Reply:
But I have adjusted to reflect that it is not a blog